ui/raidboss/data/02-arr/trial/titan-nm.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+[{
+  zoneId: ZoneId.TheNavel,
+  timelineFile: 'titan-nm.txt',
+  timelineTriggers: [
+    {
+      // Early Callout for Tank Cleave
+      id: 'TitanNm Rock Buster',
+      regex: /Rock Buster/,
+      beforeSeconds: 2,

```

Comment:
```suggestion
      beforeSeconds: 5,
```

I think 2 seconds is not much time to move out of the way or cast a heal.  The other titans say 7 seconds.  I think 5 or 7 would be fine here, but mostly arguing for something larger than 2.